css/styles.css

```diff
@@ -328,3 +328,115 @@ ul li {
   align-items: center;
   text-align: center;
 }
+
+content {
+  width: 320px;
+  height: 363px;
+  font-family: 'Inter', sans-serif;
+  font-weight: 700;
+  font-style: normal;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  color: rgba(23, 43, 77, 1);
+  margin-top: 50px;
+}
```

Comment:
It seems this style is redundant as you are targeting a `content` tag rather than a `.content` class selector. I would also recommend renaming the class name to a more description class name, `content` might be too generic.

Also, please check the CSS properties used here, many are redundant as they would have been inherited from ancestor elements.  We should also consider not giving absolute units, like px to sizes and fonts.